### ISSUE TITLE
File browser bread crumbs are not tabbable

### ISSUE THREAD

**Issue by IsabelParedes** [jupyterlab/jupyterlab#18766]
## Description

The breadcrumbs in the file browser cannot be navigated with a keyboard, they are not accessible.

<img width="715" height="448" alt="Image" src="https://github.com/user-attachments/assets/eeb2c99c-117c-4d34-ba49-6ec77e0162cc" />

## Reproduce

1. Go to File Browser panel
2. Press `Tab` or arrow keys to try to reach the bread crumbs

## Expected behavior

- The breadcrumbs should be tabbable and also the buttons that compose the breadcrumbs should be able to be activated with `Spacebar` or `Enter`

## Context

- Operating System and version: Arch Linux
- Browser and version: Chromium v146.0.7680.177
- JupyterLab version: 4.5.5
